.github/workflows/python-package.yml

```
@@ -35,4 +35,4 @@ jobs:
       run: |
         pytest
     - name: Upload Coverage to Codecov
-      uses: codecov/codecov-action@v1
+      uses: codecov/codecov-action@v2.1.0
```

```suggestion
      uses: codecov/codecov-action@v2
```
Dependabot sometimes fails to respect granularity of the semver. We should continue to only specify the major version here.